subprojects/language-java/src/integTest/groovy/org/gradle/api/tasks/compile/JavaCompileToolchainIntegrationTest.groovy

```diff
@@ -235,45 +236,7 @@ class JavaCompileToolchainIntegrationTest extends AbstractIntegrationSpec implem
         then:
         executedAndNotSkipped(":compileJava")
         outputContains("Compiling with toolchain '${otherJdk.javaHome.absolutePath}'")
-        JavaVersion.toVersion(compatibilityVersion) == JavaVersion.forClass(javaClassFile("Foo.class").bytes)
-
-        where:
-        forkOption   | configure                                       | appendPath
-        "java home"  | 'options.forkOptions.javaHome = file("<path>")' | ''
-        "executable" | 'options.forkOptions.executable = "<path>"'     | OperatingSystem.current().getExecutableName('/bin/javac')
-    }
-
-    @Issue("https://github.com/gradle/gradle/issues/22397")
-    def "uses source and target compatibility from earlier toolchain from forkOptions #forkOption"() {
```

Comment:
💬 Tests from this file moved to `JavaCompileCompatibilityIntegrationTest.groovy`